convert2rhel/unit_tests/logger_test.py

```diff
@@ -85,7 +85,7 @@ def test_log_format(self):
             logging.DEBUG, tempstr)
         self.check_formatter_result(
             logging.INFO, self.test_msg)
-        tempstr = "WARNING - %s" % self.test_msg
+        tempstr = "\x1b[93mWARNING - %s\x1b[0m" % self.test_msg
```

Comment:
We should be consistent with how we defined it in logger
```suggestion
        tempstr = "\033[93mWARNING - %s\033[0m" % self.test_msg
```